seedlet/classes/class-seedlet-custom-colors.php

```diff
@@ -293,7 +292,7 @@ function on_customize_controls_enqueue_scripts() {
 		];
 
 		wp_enqueue_script( $handle, $src, $deps );
-		wp_script_add_data( $handle, 'data', sprintf( 'var _validateWCAGColorContrastExports = %s;', wp_json_encode( $exports ) ) );
+		wp_script_add_data( $handle, 'data', sprintf( 'var seedletValidateWCAGColorContrastExports = %s;', wp_json_encode( $exports ) ) );
 
 		// Custom color contrast validation text
 		wp_localize_script( $handle, 'validateContrastText', esc_html__( 'This color combination may be hard for people to read. Try using a brighter background color and/or a darker foreground color.', 'seedlet' ));
```

Comment:
The specific feedback here was "those variables should be prefixed". 

I'm not exactly sure what variable the reviewer is referring to, but I changed the name of the JS variable here, as well as adding a prefix to the `$handle`.